src/xrpld/app/misc/CredentialsHelper.cpp

```diff
@@ -0,0 +1,204 @@
+//------------------------------------------------------------------------------
```

Comment:
nit: naming conventions would say that this file should be named `CredentialHelpers` instead of `CredentialsHelper` (I actually got a bit confused with this name being different when looking for it in my IDE).

Reply:
renamed